examples/overview/source/app.d

```diff
@@ -493,6 +498,154 @@ class SourceEditorWithSettings : Panel
     }
 }
 
+/// List controls & combo boxes
+class TabForLists : Panel
+{
+    static class State : WidgetState
+    {
+        TypeInfo_Class selectedClass;
+        TypeInfo_Class selectedDetail;
+    }
+
+    override State createState()
+    {
+        return new State;
+    }
+
+    override void build()
+    {
+        import std.array : array;
+        import std.algorithm : map, splitter;
+        import std.string : join;
+        import beamui.graphics.colors : decodeHexColor, decodeTextColor;
+
+        State st = use!State;
+
+        TypeInfo_Class[] classes;
+        dstring[] classNames;
+        foreach (m; ModuleInfo) {
```

Comment:
This is slow and can take 1-2 ms. Move these arrays into `State` and initialize in its constructor (or make them somehow static as did with `colorNames`).

A general rule is that if there are expensive calculations, they should be done somewhere outside `build`, because it is called quite often.